ui/_locales/en/messages.json

```diff
@@ -277,7 +277,7 @@
     "language": "Language",
     "bugReport": "Bug report",
     "connectedWebsites": "Connected websites",
-    "addCustomAsset": "Custom assets",
+    "addCustomAsset": "Custom QUAI assets",
```

Comment:
We've made a slight update to the title for adding a custom asset on the settings page to enhance the user experience. Does that sound good to you?

Reply:
This is fine